assets/scss/components/moj/banner/_banner.scss

```diff
@@ -0,0 +1,55 @@
+/* ==========================================================================
```

Comment:
The design uses the Banner component from the MoJ Design System. As it is such a simple component - and I think the only one we're planning - I have just copied it.

I figured this would be preferable to adding the whole MoJ library in - which we could do if we end up using more components... (Or we could just add it now - happy to change 🤷‍♂️ )

Reply:
I can see from both sides.

Including only this component means no need to import the library and set that up, but I'd make sure everywhere that code is used from that library needs a comment so it's easy to find when/if it gets replaced? The negative is (as unlikely as it is), if the component ever gets updated we won't get those updates so I'm happy to leave to you to decide 😁

Reply:
Updated to include the MoJ library just for the component we use (so far). Haven't bothered adding the JS initialisation bits as the banner component doesn't use this so no benefit at this stage.